clayui.com/src/pages/404.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayIcon from '@clayui/icon';
+import {Link} from 'gatsby';
+import React from 'react';
+
+import clayLogo from '../../static/images/home/clay_logo.svg';
+import LayoutNav from '../components/LayoutNav';
+
+const spritemap = '/images/icons/icons.svg';
+
+const PageNotFound = () => {
+	return (
+		<div className="clay-404-page-container d-flex flex-column min-vh-100">
+			<header className="d-flex">
+				<div className="container-fluid-max-xl d-flex mb-0 ml-auto mr-auto mt-0 w-100">
+					<Link className="d-none d-sm-flex mx-3" to="/">
+						<img
+							alt="Clay"
+							className="align-items-center d-flex justify-content-center"
+							src={clayLogo}
+						/>
+					</Link>
+
+					<LayoutNav />
```

Comment:
Wrap with `<div class="flex-grow-1"></div>` or `<div class="w-100"></div>`

Reply:
Adding one of these puts the `Link` and the `LayoutNav` in their own rows, it seems to work great without this wrapper.

Reply:
This wrapper puts the `Link` and the `Navbar` in their own rows, without this wrapper they're good.

Having `.navbar { width: 100% }` in css makes the version **without the wrapper you proposed** work fine.